django_spanner/client.py

```diff
@@ -9,5 +9,11 @@
 
 
 class DatabaseClient(BaseDatabaseClient):
+    """Wraps the Django base class.
+
+    TODO: Consider actual implementation of `runshell` method.
```

Comment:
Please make this an issue instead of in the docstring.

Reply:
Moved into a regular code comment for now.